CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.11.10]
```

Comment:
Should we also add a note about the scipy's bump?

Reply:
579341605b186561b4b53f1185e61c7ccaa984c9 Done!